modules/archetypes/lib/policy_assignments/policy_assignment_es_deploy_azsqldb_auditing.tmpl.json

```diff
@@ -11,15 +11,15 @@
     "description": "To ensure the operations performed against your SQL assets are captured, SQL servers should have auditing enabled. If auditing is not enabled, this policy will configure auditing events to flow to the specified Log Analytics workspace.",
     "displayName": "Configure SQL servers to have auditing enabled to Log Analytics workspace",
     "policyDefinitionId": "/providers/Microsoft.Authorization/policyDefinitions/25da7dfb-0666-4a15-a8f5-402127efd8bb",
-    "enforcementMode": null,
+    "enforcementMode": "Default",
     "nonComplianceMessages": [
       {
         "message": "SQL servers {enforcementMode} have auditing enabled to Log Analytics workspace."
       }
     ],
     "parameters": {
       "logAnalyticsWorkspaceId": {
-        "value": "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/${root_scope_id}-mgmt/providers/microsoft.operationalinsights/workspaces/${root_scope_id}-la"
+        "value": "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/${root_scope_id}-mgmt/providers/Microsoft.OperationalInsights/workspaces/${root_scope_id}-la"
```

Comment:
@matt-FFFFFF Please note that reverting this won't break the previous fix as the `lower()` overrides this. I hadn't realised that some other policies use the `logAnalyticsResourceId` parameter. Whether it is then overridden I am unsure, but figure better to revert this script change to avoid potentially unnecessary plan changes and the unknown impact of changing these parameters.